src/js/components/TabPanesComponent.jsx

```diff
@@ -56,6 +56,12 @@ var TabPanesComponent = React.createClass({
               className="btn btn-success pull-right">
                   + New App
             </Link>
+            <Link
+                to={path}
+                query={{modal: "chaos"}}
+                className="btn btn-default pull-right">
+                  Chaos
```

Comment:
If clicked on the button the disabled state should set to true.
In this component you can add/remove a listener to ```ChaosAction.on(ChaosEvents.FINISHED, onChaosFinished)``` to enable the button again.